.github/workflows/light-sdk-tests.yml

```diff
@@ -38,6 +38,11 @@ jobs:
       - name: Setup and build
         uses: ./.github/actions/setup-and-build
 
+      - name: Build CLI
+        run: |
+          source ./scripts/devenv.sh
+          npx nx build @lightprotocol/zk-compression-cli --skip-nx-cache
```

Comment:
Why do we need to build cli explicitly?

Reply:
probably a remainder of my cache debugging attempts, will remove it.

Reply:
```suggestion
          npx nx build @lightprotocol/zk-compression-cli
```